CONTRIBUTING.md

```diff
@@ -318,6 +320,92 @@ should be taken. If however, a code-owner is demonstrably shown to intentionally
 have had acted maliciously or grossly negligent, code-owner privileges may be
 stripped with no prior warning or consent from the member in question.
 
+Other potential removal criteria:
+  * Missing 3 scheduled meetings results in ICF evaluating whether the member should be
+    removed / replaced
+  * Violation of Code of Conduct
+  
 Earning this privilege should be considered to be no small feat and is by no
 means guaranteed by any quantifiable metric. It is a symbol of great trust of
 the community of this project.
+
+
+## Concept & Release Approval Process
+
+The process for how Cosmos SDK maintainers take features and ADRs from concept to release
+is broken up into three distinct stages: **Strategy Discovery**, **Concept Approval**, and 
+**Implementation & Release Approval**
+
+
+### Strategy Discovery
+
+* Develop long term priorities, strategy and roadmap for the SDK
+* Release committee not yet defined as there is already a roadmap that can be used for the time being
+
+### Concept Approval
+
+* Architecture Decision Records (ADRs) may be proposed by any contributors or maintainers of the Cosmos SDK,
+    and should follow the guidelines outlined in the
+    [ADR Creation Process](https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/PROCESS.md)
+* After proposal, a time bound period for Request for Comment (RFC) on ADRs commences
+* ADRs are intended to be iterative, and may be merged into `master` while still in a `Proposed` status
+
+**Time Bound Period**
+
+* Time bound period for individual ADR Pull Requests should not exceed 2 weeks
+* Total time bound period for an ADR to reach a decision (`ABANDONED | ACCEPTED | REJECTED`) should not exceed
+    8 weeks
+    
+If an individual Pull Request for an ADR needs more time than 2 weeks, it should be merged
+in current state `Draft` or `Proposed`, with its contents updated to summarize
+the current state of its discussion.
+
+If an ADR is taking longer than 8 weeks to reach a final conclusion, the **Concept Approval Committee** 
+should convene to rectify the situation by either making changes to the Concept Approval
+Process (as outlined here), or making changes to the members of the Approval Committee.
+
+**Approval Committee & Decision Making**
+
+In absense of general consensus, decision making requires ⅔ vote from the three members
+of the **Concept Approval Committee**.
+
+**Committee Members**
+
+* Core Members: **Aaron** (Regen), **Bez** (Fission), **Alessio** (AiB)
+* Secondary pool of candidates to replace / substitute:
+    * **Chris Goes** (IG), **Sunny** (Sikka)
```

Comment:
how about having 5 people from the beginning and setting threshold = 3?

Reply:
I thought that the initial ADR committee was deliberately designed to be small to keep latency low. I still believe that that is a good scheme, or maybe something changed since then? And I love the idea of having a secondary pool of highly reputable and experienced SDK engineers. This looks good as it is in my opinion.

Reply:
The discussion around 2/3 was decided previously by ICF with input from the cosmos multi-entity community i believe. This was first all articulated and consensus reached in a [google doc](https://docs.google.com/document/d/1GkVMUyOY6RmFGBjG0ZkBu_Y7wDunYWukkEjFOsejqYA/edit?ts=5eede440#). I think we should stick as is for now, and if there are problems, the committee + ICF can evaluate an alternative.

Reply:
I agree with Cory here - that was the original and we should stick to it. Though I like @robert-zaremba's proposal very much indeed:

> how about having 5 people from the beginning and setting threshold = 3?

IMO that should be the target state. In Debian, the Technical Committee has a chair who votes only if there is a tie.
I think we should revisit this structure in a few months time :+1: